testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -57,13 +63,22 @@ def removeFile(fileName):
 print "running", ' '.join(args)
 subprocess.check_call(args, env=os.environ.copy())
 
-print 'Step 6. Injecting bathymetry'
-args = ['./inject_bathymetry.py',
-        'base_mesh.nc']
-print "running", ' '.join(args)
-subprocess.check_call(args, env=os.environ.copy())
+if cl_args.inject_bathymetry:
+  print 'Step 6. Injecting bathymetry'
+  args = ['./inject_bathymetry.py',
+          'base_mesh.nc']
+  print "running", ' '.join(args)
+  subprocess.check_call(args, env=os.environ.copy())
+
+if cl_args.inject_cullCell > 0.0:
```

Comment:
@sbrus89, do we need another step here to make sure that we have the bathymetry in the file after it is culled?  This is something to check.  

@pwolfram to check later in the review.

Reply:
This should be taken care of now since adding a call to inject_bathymetry.py in the cull mesh step.